app/src/main/java/fr/free/nrw/commons/contributions/MainActivity.java

```diff
@@ -165,15 +165,16 @@ public void onCreate(Bundle savedInstanceState) {
              * so that location in the EXIF metadata of the images shared by the user
              * is retained on devices running Android 10 or above
              */
-            if (VERSION.SDK_INT >= VERSION_CODES.Q) {
-                PermissionUtils.checkPermissionsAndPerformAction(
-                    this,
-                    () -> {
-                    },
-                    R.string.media_location_permission_denied,
-                    R.string.add_location_manually,
-                    permission.ACCESS_MEDIA_LOCATION);
-            }
+//            if (VERSION.SDK_INT >= VERSION_CODES.Q) {
```

Comment:
Is this no longer required? If yes, could you please remove it?

Reply:
**`ACCESS_MEDIA_LOCATION`** is required to read the location information from the pictures. But, I don't know when this permission should be invoked. 
So, I just commented that to prevent invoking it on the App's startup, the app should only ask for permission when needed. I just forgot about this permission, thank you for looking at it :+1: 
Do you know at which place the app tries to read the picture's location?

Reply:
Any time we upload any image, we require this permission. We also ask to access storage and images in the very beginning even though that's required when users use normal/custom selector.

Reply:
Well in this PR, the storage permission is asked from the user when the user tries to open `custom selector`. However, look at this documentation:-
https://developer.android.com/training/data-storage/shared/media#location-info-photos

Does it mean, we don't need ACCESS_MEDIA_LOCATION permission to read photo's location.

Reply:
Location information probably got stripped off😞
I'm currently testing this PR. I allowed access all from my Gallery and got the no location found pop up. The edit location button shows a green tick, though.

Reply:
I am making a minor change to ask for `ACCESS_MEDIA_LOCATION` permission along with other storage-related permissions. Please test the behavior again. Give me 5 mins only.